src/openrct2/paint/Paint.h

```diff
@@ -113,7 +113,10 @@ struct tunnel_entry
     uint8_t type;
 };
 
-#define MAX_PAINT_QUADRANTS 512
+// NOTE: This should be preferably a prime number. This is the amount of
```

Comment:
prime or power of 2? cause it will be much slower if it needs to modulo everything i would presume.

Reply:
Well the benchmark proves otherwise, hash maps typically use primes and not power of two for a reason, better distribution out weights the modulo performance.

Reply:
Yes but did you compare without the clamp and modulo 512?

Reply:
Yes its slightly better without changing the constant, but modifying the constant squeezes another 5-6 FPS on bpb

Reply:
It would be even best if every zoom level had a different constant to get the best out of it. But still better

Reply:
Ahh yeah im forgetting the bigger picture here in that this is just for assigning the bucket and with a prime we get better bucket distribution which ultimately would reduce other comparisons.